assets/app/scripts/services/applicationGenerator.js

```diff
@@ -56,7 +56,7 @@ angular.module("openshiftConsole")
             {
               containerPort: parseInt(parts[0]), 
               name: input.name + "-" + parts[1] + "-" + parts[0],
-              protocol: parts[1]
```

Comment:
This seems ok for TCP/UDP, but I'm not sure it will always be valid. Add a todo to revisit